app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
+        if "vaccine" not in vstr.keys() or vstr["vaccine"] == None:
+            raise NotVaccinatedError("not vaccinated")
+        elif vstr["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("outdated")
+        elif not vstr["wearing_a_mask"]:
+            raise NotWearingMaskError("not wearing mask")
```

Comment:
You should provide more descriptive error messages when raising exceptions. This helps in debugging and understanding the cause of the error
```suggestion
            raise NotVaccinatedError("Visitor is not vaccinated")
        elif vstr["vaccine"]["expiration_date"] < datetime.date.today():
            raise OutdatedVaccineError("Vaccine is outdated")
        elif not vstr["wearing_a_mask"]:
            raise NotWearingMaskError("Visitor is not wearing a mask")
```